src/cratedb/mod.rs

```diff
@@ -0,0 +1,130 @@
+use std::{borrow::Cow, collections::HashMap};
+
+use testcontainers::{core::WaitFor, Image};
+
+const NAME: &str = "crate";
+const TAG: &str = "5.10.6";
+
+/// Represents a CrateDB docker instance.
+/// It is based on the official [`crate` docker image](https://hub.docker.com/_/crate)
+///
+/// Defaults are not modified:
+/// - `super_user`: `crate`
+/// - `password`: <empty>
+/// - `pg_port`: `5432`
+/// - `http_port`: `4200`
+///
+/// You can connect run instructions by sending an HTTP call to /_sql in port 4200.
+/// [`crate http interface`]
+///
+/// or using the postgres wire compatible protocol at port 5432.
+///
+/// # Example of postgres wire protocol.
+/// ```rust
+/// use testcontainers_modules::testcontainers::runners::SyncRunner;
```

Comment:
Actually (I should Have scrolled down more), this is the fix for CI. Sorry

```suggestion
/// use testcontainers_modules::{cratedb, testcontainers::runners::SyncRunner};
```